components/scream/src/control/tests/ad_tests.cpp

```diff
@@ -31,15 +32,14 @@ TEST_CASE ("group_requirements","[!throws]")
   // Init and run a single time step
   util::TimeStamp init_time(2000,1,1,0,0,0);
   ad.initialize(atm_comm,ad_params,init_time);
-  // Verify that after initialization field "E" matches field "A"
+
+  // Verify that the atm proc group has 1 required field and 0 required groups.
+  const auto& apg = ad.get_atm_processes();
+  REQUIRE (apg->get_num_processes()==3);
+  REQUIRE (apg->get_fields_in().size()==1);
+  REQUIRE (apg->get_groups_in().size()==0);
+
   auto field_mgr = ad.get_ref_grid_field_mgr();
-  const auto& view_A = field_mgr->get_field("A").get_view<const Real**, Host>();
```

Comment:
I'm sure there is a good reason, but why do we remove this test?

Reply:
I just wanted to keep the test small, I guess, so I removed a couple of fields. But I can see there's value in checking initialization procedures. I'll bring some stuff back.